tests/test_tables.cpp

```diff
@@ -354,6 +354,22 @@ TYPED_TEST(TableSizeTwo, DeleteEntryHandleUpdate) {
   ASSERT_EQ(MatchErrorCode::EXPIRED_HANDLE, rc);
 }
 
```

Comment:
Please add a quick comment here pointing to your Github issue: https://github.com/p4lang/behavioral-model/issues/549